ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -313,6 +313,39 @@ class SignalSpec: QuickSpec {
 				observer.sendNext(1)
 				expect(values) == [1]
 			}
+
+			//  TODO: remove when the method is marked unavailable
+			it("receives next values with erroring signal") {
```

Comment:
I added 2 tests (one for `SignalProducer` and one for `Signal`) to make sure that we don't break compatibility for now.